debian/watch

```diff
@@ -0,0 +1,8 @@
+# See uscan(1) for format
+
+# Compulsory line, this is a version 4 file
+version=4
+
+# Direct Git
+opts="mode=git" https://github.com/named-data/NFD.git \
```

Comment:
nope, we still get the upstream sources from there, ideally we should patch those if we have a discrepacy.